src/controllers/challenge.controllers.js

```diff
@@ -37,40 +37,47 @@ export const getChallengeById = async (req, res) => {
   }
 };
 
-// Add a member to a challenge
-export const addMember = async (req, res) => {
+// Remove member from challenge
+export const removeMember = async (req, res) => {
   try {
-    const { id } = req.params;
-    const { user_id } = req.body;
+    const { id, userId } = req.params;
     const challenge = await Challenge.findById(id);
     if (!challenge) {
```

Comment:
let's put a challenge status validation. Joiners shouldn't be able join closed challenges.

Reply:
Addressed here: https://github.com/LangaraBeetles/weaup-api/pull/15/commits/7213865e1dabf4682d53d19c46c0fe67ea1f1fd8.

Reply:
Sorry for the copy + pasted review, but you got what I was referring to 😄  Looks good!